main.go

```diff
@@ -13,5 +15,7 @@ func main() {
 		},
 	)
 
+	cli.SubCommand("migrate create", migration.Migrate)
```

Comment:
shouldn't this command be `gofr-migrate` create.... ?

Reply:
gofr would be included in the tool name, so command should not have it.